sql/hive/src/test/scala/org/apache/spark/sql/sources/BucketedReadSuite.scala

```diff
@@ -59,6 +60,77 @@ class BucketedReadSuite extends QueryTest with SQLTestUtils with TestHiveSinglet
     }
   }
 
+  test("read partitioning bucketed tables with bucket pruning filters") {
```

Comment:
Yeah. Thank you! This answers my question how to verify if the bucket pruning works.